### PR TITLE
feat(StatusLabel): added status label component

### LINKED ISSUE #44: Background is not white
So, in the design files, the background is `#FFFFFF` (white), but in the design system we have a color called `background` which is `#f6f6f9`. Which is correct, @AnneMatilde ?

### PR DESCRIPTION
**Pull request by TanyaDolgopolova**
fixed color names
Close #44

### CODE REVIEW

**Review by cobraz — APPROVED** [review]

**Comment by cobraz** [comment]
@TanyaDolgopolova Let me know when you've fixed the conflicts :)

**Comment by TanyaDolgopolova** [comment]
@cobraz merge conflict - fixed 😌

File: src/components/Field/Field.tsx [hunk 1]
```diff
@@ -2,12 +2,16 @@ import React from 'react';
...
+
 export const Field: ForwardRef<
   HTMLDivElement,
-  FieldProps<any>
-> = React.forwardRef(({ as: Control = Input, label, name, ...props }, ref) => {
+  FieldsProps
+> = React.forwardRef(({ as: Control = Input, label, name, alignItems = 'center', ...props }, ref) => {
   return (
-    <Flex {...getMargin(props)} sx={{ alignItems: 'flex-start' }}>
+    <Flex {...getMargin(props)} sx={{ alignItems: alignItems }}>
```
Comment (kvokov) [inline src/components/Field/Field.tsx:14]: may be shorter `sx={{ alignItems }}`

File: src/components/StatusLabel/StatusLabel.stories.tsx [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+import { StatusLabel } from './StatusLabel';
+
+export default {
+  title: 'StatusLabel',
+  component: StatusLabel,
+  parameters: { actions: { argTypesRegex: '^on.*' } },
+};
+
+export const activeStatusLabel = (props: any) => (
+  <StatusLabel label="Active" active={true} {...props} />
```
Comment (kvokov) [inline src/components/StatusLabel/StatusLabel.stories.tsx:11]: true boolean prop passing just as:
 `<StatusLabel label="Active" active {...props} />`

**Review by AnneMatilde — APPROVED** [review]

File: src/components/Field/Field.tsx (hunk 1 above)
Comment (cobraz) [inline src/components/Field/Field.tsx]:
```suggestion
    <Flex {...getMargin(props)} sx={{ alignItems }}>
```

**Review by cobraz — APPROVED** [review]
LGTM 🎉

**Review by cobraz — APPROVED** [review]

**Review by kvokov — APPROVED** [review]
👍

**Pull request merged**